x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/create_security_rule_type_wrapper.ts

```diff
@@ -523,6 +527,7 @@ export const createSecurityRuleTypeWrapper: CreateSecurityRuleTypeWrapper =
                   indexingDurations: result.bulkCreateTimes,
                   enrichmentDurations: result.enrichmentTimes,
                   executionGap: remainingGap,
+                  gapRange: experimentalFeatures?.storeGapsInEventLogEnabled ? gap : undefined,
```

Comment:
I see a couple of other calls to `ruleExecutionLogger.logStatusChange()` in this file where the `metrics` are reported, but the new `gapRange` prop is not passed.

Is it not necessary to pass it in all logic branches of rule execution? At least in the ones wher the `newStatus` is `failed`.